File: Pset6/Sentimental-Mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Please insert height of pyramid: ")
    if (height > 0 and height < 9):
        break

for i in range(height):
    print(" " * (height - i - 1), end="")
    print("#" * (i + 1), end="")
    print("  ", end="")
    print("#" * (i + 1), end="")
    print(" " * (height - i - 1))

File: Pset1/Mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // ask for height and store in int variable
    int height;

    do
    {
        height = get_int("Please insert height of pyramid: ");
    }
    while (height < 1 || height > 8);

    // draw pyramid
    for (int i = 1; i <= height; i++)
    { 

        // draw spaces
        for (int j = height - i; j > 0; j--)
        {
            printf(" ");
        }

        // draw hashtags
        for (int k = 1; k <= i; k++)
        {
            printf("#");
        }    
        printf("\n"); // new line after each row
    }

}
